apps/src/componentLibrary/tooltip/Tooltip.tsx

```diff
@@ -60,31 +62,39 @@ export const TooltipOverlay: React.FunctionComponent<TooltipOverlayProps> = ({
  * Design System: Tooltip Component.
  * Renders Tooltip for a given html element (Needs to be wrapped in TooltipOverlay, see stories or README.md).
  */
-const Tooltip: React.FunctionComponent<TooltipProps> = ({
-  text,
-  tooltipId,
-  iconLeft,
-  iconRight,
-  direction = 'onTop',
-  className,
-  size = 'm',
-}) => {
-  return (
-    <div
-      id={tooltipId}
-      className={classnames(
-        moduleStyles.tooltip,
-        moduleStyles[`tooltip-${direction}`],
-        moduleStyles[`tooltip-${size}`],
-        className
-      )}
-      role="tooltip"
-    >
-      {iconLeft && <FontAwesomeV6Icon {...iconLeft} />}
-      <span className={moduleStyles.tooltipText}>{text}</span>
-      {iconRight && <FontAwesomeV6Icon {...iconRight} />}
-    </div>
-  );
-};
+const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(
```

Comment:
Should we follow the pattern you've established previously to mark these as for internal use only and name it `_Tooltip`? I see you said in the docs that this could technically be used directly, but maybe we should just not advise people to use it directly?

Reply:
Yeah, was think about this also. Thought it might be better to keep it 'public' just because other internal components are internal cause they are not designed for any usage outside of Design System, while `Tooltip` can theoretically be used if user wants to create some custom positioned tooltip. At the same time, recommended way to use it is, of course, `WithTooltip`.


Though, now that you're asking about it, I'm now hesitant about leaving `Tooltip`  public

Reply:
I'd vote to make it not public (ie, drive usage towards `WithTooltip` / away from custom implementations if possible), but up to you :)